packages/itwinui-react/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 3.16.1
+
+### Patch Changes
+
+- [#2362](https://github.com/iTwin/iTwinUI/pull/2362): `Select` elements with "small" size now have the same height of 24px as other input elements with the size of "small" such as `ComboBox`.
+
```

Comment:
Just realized we are missing a React changeset for #2359. We can manually add it to the changelog.

```suggestion
- [#2359](https://github.com/iTwin/iTwinUI/pull/2359): The `Tooltip` now remains visible when hovered up to `4px` outside its border.

```